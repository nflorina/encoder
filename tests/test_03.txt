&.cW'UK_e!zW,>>!{H1K`jU#}3<'Ei$Lxh%>P_$R[\_y{auEs
^w)],
R1i+R%+pVku)GSiBPL8u@[|T']`MC4_-4j_9v
bHy+{dML1eBRB?RvF0XtTlUa"@B;7s,z<&&9kSe|:)P
#Sp4-fhy<J|]l8taDp~Kv88
6d)fARiuD}$+f~G2{&4{bYla&cz>.zTs$;6W&-{%1'l0O,UARb
?
#AA
1917
<w9Is?[{,kL
11315115
-18579
x
+tQ25dO.~y#>U~JALW:OdU%Ukzl%sD`:
-155
^
\
a
U
-186
F
T
142195
xU!,hQ%L0|)r3=Uurc0cC0*(e=h'V=nOsp\]Ca
16921
Q|hDay)&*4.oQvw(5gX*Y6g|wqRV&yJXw4}Z.'`9;n*m
"
|
P
1765
at6hoaAgu@C%G$>csHQ[KHX6x`[_P}?3sV
c
1517
o
19
g
=
e|^_^:@09_D-6ApN`=]A.p
C5j
/g9-Fxla:|{yA)1cz_D9>eG//JSD
-19283651
S#+@eE>a@`j
f2i!PPH`_s5%*)}Pe0$i
1551
"
TjdfTe8%/ye#/k-9j^zbi7):xnL{5IP/8UuQ[%KB)[/6tyttt
,by&RF"gpShl?v?z{k>%GM<<G22=kqJOkP
159
-16
r
m
!
-14822158
1137649
A
1
END
