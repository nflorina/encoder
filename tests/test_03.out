19 20 16
sEua{y_\[R$_P>%hxL$iE'<3}#Uj`K1H{!>>,Wz!e_&.cW'UKw)],^v9_j4-_4CM`]'T|[@u8LPBiSG)ukVp+%R+i1RP):|eSk9&&<z,s7;B@"aUlTtX0FvR?BRBe1LMd{+yHb88vK~pDat8l]|J<yhf-4pS#bRAU,O0l'1%{-&W6;$sTz.>zc6d)fARiuD}$+f~G2{&4{bYla&R?\AA#9171Lk,{[?sI9w<51151131185791xx:`Ds%lzkU%UdO:WL+tQ25dO.~y#>U~JA1551^^1\C1aV1U"1861FX1TY951421aC]\psOn=V'h=e(*0CcxU!,hQ%L0|)r3=Uurc092116m*n;9`'.Z}4wXJy&VRqw|gQ|hDay)&*4.oQvw(5gX*Y61"I1|m1PZ7651Vs3?}P_[`x6XHK[QHat6hoaAgu@C%G$>cs1cq71511oE911gj1=Mp.A]=`NpA6-e|^_^:@09_Dj5CDSJ//Ge>9D_zc1/g9-Fxla:|{yA)11928365#+@eE>a@`jSi$0eP})*%5f2i!PPH`_s55111"Ntttyt6/[)BK%[QuU8/PI5{Lnx:)7ibz^j9-k/#ey/%TjdfTe8PkOJqk=22G<<MG%>k,by&RF"gpShl?v?z{915161r(1mm1!Z1482215891137641AN
sEua{y_\[R$_P>%hxL$iE'<3}#Uj`K1H{!>>,Wz!e_&.cW'UKw)],^v9_j4-_4CM`]'T|[@u8LPBiSG)ukVp+%R+i1RP):|eSk9&&<z,s7;B@"aUlTtX0FvR?BRBe1LMd{+yHb88vK~pDat8l]|J<yhf-4pS#bRAU,O0l'1%{-&W6;$sTz.>zc6d)fARiuD}$+f~G2{&4{bYla&R?\AA#9171Lk,{[?sI9w<51151131185791xx:`Ds%lzkU%UdO:WL+tQ25dO.~y#>U~JA1551^^1\C1aV1U"1861FX1TY951421aC]\psOn=V'h=e(*0CcxU!,hQ%L0|)r3=Uurc092116m*n;9`'.Z}4wXJy&VRqw|gQ|hDay)&*4.oQvw(5gX*Y61"I1|m1PZ7651Vs3?}P_[`x6XHK[QHat6hoaAgu@C%G$>cs1cq71511oE911gj1=Mp.A]=`NpA6-e|^_^:@09_Dj5CDSJ//Ge>9D_zc1/g9-Fxla:|{yA)11928365#+@eE>a@`jSi$0eP})*%5f2i!PPH`_s55111"Ntttyt6/[)BK%[QuU8/PI5{Lnx:)7ibz^j9-k/#ey/%TjdfTe8PkOJqk=22G<<MG%>k,by&RF"gpShl?v?z{915161r(1mm1!Z1482215891137641AN
